docs/instruments.md

```diff
@@ -3,25 +3,22 @@
 ## Create Instrument
 
 To create a new validated instrument, execute the appropriate one-liner using
-the correct instrument name.
-The initial status of the instrument is 'wash required'.
+the correct instrument names. The value of the `external_instrument` field is
```

Comment:
Should that be `external_name`?

Reply:
Thanks, fixed in https://github.com/wtsi-npg/npg_tracking/pull/714/commits/1eb4e89f11305d3a8c3d87ec8f761d29244c0313